tags/state/test/test.server.js

```diff
@@ -0,0 +1,69 @@
+const { render } = require("@marko/testing-library");
+const { expect } = require("chai");
+const simple = require("./fixtures/simple.marko");
+const multi = require("./fixtures/multi.marko");
+const assignment = require("./fixtures/assignment.marko");
+const bool = require("./fixtures/boolean.marko");
+const noInit = require("./fixtures/noInitializer.marko");
+
+describe("server", () => {
+  it("renders as usual without state tag", async () => {
+    const { container, getByText } = await render(simple, {
+      value: "Custom state text"
+    });
+
+    expect(container)
+      .has.property("children")
+      .with.length(1);
+
+    expect(getByText("Custom state text")).to.exist;
+  });
+
+  it("renders as multi state tags", async () => {
+    const { container, getByText } = await render(multi, {
+      value: "Custom state text"
+    });
+
+    expect(container)
+      .has.property("children")
+      .with.length(1);
+
+    expect(getByText("Custom state text")).to.exist;
+    expect(getByText("2")).to.exist;
+  });
+  it("renders as using complex functions", async () => {
+    const { container, getByText } = await render(assignment);
+
+    expect(container)
+      .has.property("children")
+      .with.length(1);
+
+    expect(getByText("1")).to.exist;
```

Comment:
`.to.exist` on `getByText` is a noop since `getBy` apis already through. This should probably be `queryByText` like I was saying before.